src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp

```diff
@@ -134,19 +135,19 @@ namespace picongpu
 
```

Comment:
ah there is even more here...

well well, I can probably even cut the code of these files by half with todays wrappers we have.